core/server/src/main/java/alluxio/worker/DefaultAlluxioWorker.java

```diff
@@ -243,6 +246,7 @@ public void stop() throws Exception {
   }
 
   private void startWorkers() throws Exception {
+    mBlockWorker.init(getAddress());
```

Comment:
I don't think this init makes sense, but its not something you need to address in this PR.